site/src/pages/docs/client-factory-builder.mdx

```diff
@@ -0,0 +1,131 @@
+# Building a client using ClientFactoryBuilder
+
+A <type://ClientFactoryBuilder> is a builder class that enables you to create a client factory
+which is able to customize various parameters. For example, some people want to make longer connection timeout.
+If so, the builder provides connectTimeoutMillis to change from the default value to what you want to change.
+
+You can simply create <type://ClientFactoryBuilder> as following example:
+
+```java
+final ClientFactory factory =
+          ClientFactory.builder()
+                       // Set the connection timeout to 5 seconds.
+                       .connectTimeoutMillis(5000)
+                       // Disable certificate verification; never do this in production!
+                       .tlsNoVerify()
+                       .build();
```

Comment:
It would be better if we add:
- Use try-with-resource so that users learn that a `ClientFactory` should be closed to prevent a leak.
- How to set the created `ClientFactory` using `WebClientBuilder` and `ClientBuilder`
- Add an example for `ClientFactoryOption`?

Reply:
It would be a good idea to make users implicitly enforce to implement in a safe manner.